api.json

```diff
@@ -1738,7 +1738,7 @@
            "example": 12
           },
          "message": {
-           "description":"Message is a network-specific error message.",
+           "description":"Message is a network-specific error message.  The message MUST NOT change for a given code. In particular, this means that any contextual information should be included in  the details field.",
```

Comment:
[nit] extra spaces:
* `message.  The` -> `message. The`
*  `included in  the` -> `included in the`